src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -134,16 +138,21 @@ void Graph::Replicate(const std::shared_ptr<const ov::Model> &model) {
 
         AddNode(node);
         if (op->get_type_info() == op::v0::Parameter::get_type_info_static()) {
-            const std::string name = get_port_name(ov::Output<ov::Node>(op, 0));
-            inputNodesMap[name] = node;
+            auto input_index = model->get_parameter_index(std::dynamic_pointer_cast<op::v0::Parameter>(op));
+            OPENVINO_ASSERT(input_index >= 0,
+                            "Can not find op: ",
+                            op->get_friendly_name(),
+                            " in model parameter list!");
```

Comment:
```suggestion
              OPENVINO_ASSERT(input_index >= 0,
                            "CPU plugin cannot find op: ",
                            op->get_friendly_name(),
                            " in model parameter list!");                         
```

Reply:
If possible, let add "CPU plugin" to exception message to make it is more friendly UX to user.